packages/react/src/components/Markdown/Markdown.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import './Markdown.css';
-import { MessageEmoji } from '@rocket.chat/fuselage';
+//import { MessageEmoji } from '@rocket.chat/fuselage';
+import {MessageEmoji} from '../EmojiPicker/EmojiPicker'
```

Comment:
We do not export MessageEmoji from EmojiPicker. It will cause errors. 
MessageEmoji is used to wrap big emoji messages. That is when we send a single emoji as message, it appears to be slightly bigger in size.

Reply:
Hi @abhinavkrin , thanks for your input. I'll fix that ASAP